OtripleS.Portal.Web.Tests.Unit/Services/StudentViews/StudentViewServiceTests.Exceptions.RetrieveAll.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using OtripleS.Portal.Web.Models.StudentViews;
+using OtripleS.Portal.Web.Models.StudentViews.Exceptions;
+using Xunit;
+
+namespace OtripleS.Portal.Web.Tests.Unit.Services.StudentViews
+{
+    public partial class StudentViewServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(DependencyExceptions))]
+        public async Task ShouldThrowStudentViewDependencyExceptionIfDependencyErrorOccursAndLogItAsync(
+           Exception dependencyException)
+        {
+            // given
+            var expectedStudentViewDependencyException =
+                new StudentViewDependencyException(dependencyException);
+
+            this.studentServiceMock.Setup(service =>
+                service.RetrieveAllStudentsAsync())
+                    .ThrowsAsync(dependencyException);
+
+            // when
+            ValueTask<List<StudentView>> retrieveAllStudentsTask =
```

Comment:
Rename the variable to retrieveAllStudentViewsTask

Do verify in other tests too.

Reply:
retrievedStudentViewsTask sounds good?

Reply:
@ysnarafat look at the method name - match with that. plus `Task`

Reply:
@hassanhabib Okay.